### PR TITLE
docs: Fix typos identified using codespell

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

**Comment by jcfr** [comment]
@floryst @jourdain  Let me know if you think `fix: Fix typo in ivar identified using codespell` should be categorized as a breaking change or not.

File: Sources/Rendering/Core/Camera/index.js [hunk 1]
```diff
@@ -380,7 +380,7 @@ function vtkCamera(publicAPI, model) {
     let t = thickness;
     if (t < 1e-20) {
       t = 1e-20;
-      vtkDebugMacro('Thickness is set to minimum.');
+      vtkDebugMacro('Thi./Sources/Rendering/Core/Camera/index.jsckness is set to minimum.');
```
Comment (finetjul) [inline Sources/Rendering/Core/Camera/index.js]: looks odd to me

Comment (jcfr) [reply Sources/Rendering/Core/Camera/index.js]: Good catch

Comment (jcfr) [reply Sources/Rendering/Core/Camera/index.js]: I don't have access to my dev computer (at the airport now), is it something you could help address ? If not, I will take care of it when I return on March 2nd.

**Comment by jourdain** [comment]
This could even be a `chore(spell): Update doc and iVar` as no API change is truly happening.

**Comment by jourdain** [comment]
Was fixed in https://github.com/Kitware/vtk-js/pull/1344

**Pull request closed without merging**